includes/admin/meta-boxes/views/subscription-scheme.php

```diff
@@ -11,7 +11,10 @@
 }
```

Comment:
Did you mean to change permissions on this file (from 755 to 644) or was that accidental?